src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1011,10 +1011,23 @@ else if (fieldClass == URI.class) {
             result = "URI";
         }
         else if (fieldClass == Pattern.class) {
-            result = "Regular Expression";
+            if ("SuppressionSingleFilter:checks".equals(checkProperty)
+                || "SuppressionXpathSingleFilter:files".equals(checkProperty)
+                || "IllegalTokenText:format".equals(checkProperty)) {
```

Comment:
same here, why it is an exception/

Reply:
while discussion is not finished on how to deal with this Checks, lets keep them as they were on  "Regular Expression" (not real java type), See details in telegram group.